src/ui.form-date-picker-range/index.vue

```diff
@@ -1009,26 +1040,27 @@ function onMouseoverCalendar() {
 
   if (isRangeInputFocus || !rangeInputStartRef.value || !rangeInputEndRef.value) return;
 
-  if ((rangeStart.value && rangeEnd.value && !inputRangeEndError.value) || !rangeStart.value) {
+  if (
+    (calendarInnerValue.value.from && calendarInnerValue.value.to && !inputRangeEndError.value) ||
```

Comment:
`const hasValues = calendarInnerValue.value.from && calendarInnerValue.value.to`

make if statement more readable